go.mod

```diff
@@ -23,9 +19,9 @@ require (
 	github.com/onsi/gomega v1.27.6
 	github.com/pivotal-cf-experimental/gomegamatchers v0.0.0-20180326192815-e36bfcc98c3a
 	github.com/rubenv/sql-migrate v1.4.0
-	github.com/tedsuo/ifrit v0.0.0-20230330192023-5cba443a66c4
+	github.com/tedsuo/ifrit v0.0.0-20230516164442-7862c310ad26
 	github.com/tedsuo/rata v1.0.0
-	github.com/vishvananda/netlink v1.1.0
+	github.com/vishvananda/netlink v1.2.1-beta.2
```

Comment:
Are we sure we want to use a beta version here?

Reply:
Unfortunately, yes. netlink v1.1.0 isn't compatible with the newer github.com/containernetworking/plugins library.